doc/source/notebooks/theory/upper_bound.ipynb

```diff
@@ -322,8 +331,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Lower bound: -65.059355\n",
-      "Upper bound: -65.058996\n"
+      "Lower bound: -62.487148\n",
+      "Upper bound: -62.479897\n"
```

Comment:
do we know why it changed ? are we not setting the random seed? would be good to have notebooks be reproducible ...